application/services/Reports/getTbAllSitesResultsSheet.sql

```diff
@@ -0,0 +1,860 @@
+    flattenedevaluationresults.`1 - mtb`,
+    flattenedevaluationresults.`1 - rif`,
+    flattenedevaluationresults.`1 - probe 1`,
+    flattenedevaluationresults.`1 - probe 2`,
+    flattenedevaluationresults.`1 - probe 3`,
+    flattenedevaluationresults.`1 - probe 4`,
+    flattenedevaluationresults.`1 - probe 5`,
+    flattenedevaluationresults.`1 - probe 6`,
+    flattenedevaluationresults.`2 - DATE tested`,
+    flattenedevaluationresults.`2 - instrument serial`,
+    flattenedevaluationresults.`2 - instrument last calibrated`,
+    flattenedevaluationresults.`2 - mtb`,
+    flattenedevaluationresults.`2 - rif`,
+    flattenedevaluationresults.`2 - probe 1`,
+    flattenedevaluationresults.`2 - probe 2`,
+    flattenedevaluationresults.`2 - probe 3`,
+    flattenedevaluationresults.`2 - probe 4`,
+    flattenedevaluationresults.`2 - probe 5`,
+    flattenedevaluationresults.`2 - probe 6`,
+    flattenedevaluationresults.`3 - DATE tested`,
+    flattenedevaluationresults.`3 - instrument serial`,
+    flattenedevaluationresults.`3 - instrument last calibrated`,
+    flattenedevaluationresults.`3 - mtb`,
+    flattenedevaluationresults.`3 - rif`,
+    flattenedevaluationresults.`3 - probe 1`,
+    flattenedevaluationresults.`3 - probe 2`,
+    flattenedevaluationresults.`3 - probe 3`,
+    flattenedevaluationresults.`3 - probe 4`,
+    flattenedevaluationresults.`3 - probe 5`,
+    flattenedevaluationresults.`3 - probe 6`,
+    flattenedevaluationresults.`4 - DATE tested`,
+    flattenedevaluationresults.`4 - instrument serial`,
+    flattenedevaluationresults.`4 - instrument last calibrated`,
+    flattenedevaluationresults.`4 - mtb`,
+    flattenedevaluationresults.`4 - rif`,
+    flattenedevaluationresults.`4 - probe 1`,
+    flattenedevaluationresults.`4 - probe 2`,
+    flattenedevaluationresults.`4 - probe 3`,
+    flattenedevaluationresults.`4 - probe 4`,
+    flattenedevaluationresults.`4 - probe 5`,
+    flattenedevaluationresults.`4 - probe 6`,
+    flattenedevaluationresults.`5 - DATE tested`,
+    flattenedevaluationresults.`5 - instrument serial`,
+    flattenedevaluationresults.`5 - instrument last calibrated`,
+    flattenedevaluationresults.`5 - mtb`,
+    flattenedevaluationresults.`5 - rif`,
+    flattenedevaluationresults.`5 - probe 1`,
+    flattenedevaluationresults.`5 - probe 2`,
+    flattenedevaluationresults.`5 - probe 3`,
+    flattenedevaluationresults.`5 - probe 4`,
+    flattenedevaluationresults.`5 - probe 5`,
+    flattenedevaluationresults.`5 - probe 6`,
+    flattenedevaluationresults.`comments`,
+    flattenedevaluationresults.`comments FOR reports`,
+    flattenedevaluationresults.`1 - score`,
+    flattenedevaluationresults.`2 - score`,
+    flattenedevaluationresults.`3 - score`,
+    flattenedevaluationresults.`4 - score`,
+    flattenedevaluationresults.`5 - score`
+-- --------------------------------------- START NON-PTCC COORDINATOR FIELDS ------------------------------------------
```

Comment:
Please move this up by 5 lines. PTCCs shouldn't be presented with any scores

Reply:
Oh ok, so all scores not just fin score, will do